client-java/src/component-tests/java/uk/gov/gchq/palisade/client/test/component/PalisadeServiceTest.java

```diff
@@ -57,13 +60,13 @@ void testSubmit() throws Exception {
         var palisadeResponse = service.submit(palisadeRequest);
 
         assertThat(palisadeResponse)
-            .as("A valid response has been returned")
+            .as("check valid PalisadeResponse")
```

Comment:
checking something is not null does not check that it is valid, update the description

Reply:
I've tidied that test up a bit.